lib/Xmds/Soap.php

```diff
@@ -423,6 +429,22 @@ protected function doRequiredFiles($serverKey, $hardwareKey, $httpDownloads)
 
         // Create a comma separated list to pass into the query which gets file nodes
         $layoutIdList = implode(',', $layouts);
+        if (isset($this->display->overrideConfig))
```

Comment:
refactor - should all be replace-able with `$display->getSetting('versionMediaId');` ?